src/shared/components/technologiesList/technologiesList.component.tsx

```diff
@@ -40,63 +21,45 @@ interface TechnologiesListProps {
 }
 
 export const TechnologiesList = ({ technologies, emptyResults, rings, hasNoAreaSelected }: TechnologiesListProps) => {
-  const [hoveredItem, setHoveredItem] = useState<string | null>(null);
+  const intl = useIntl();
   const searchText = useSelector(selectSearch);
   const [scrollTopReached, setScrollTopReached] = useState(true);
   const [scrollBottomReached, setScrollBottomReached] = useState(false);
-  const isTouchDevice = Boolean('ontouchstart' in window || navigator.maxTouchPoints);
   const dispatch = useDispatch();
   const handleOpenPopup = (technologyId: TechnologyId) => dispatch(openTechnologyPopup(technologyId));
 
   const sortedTechnologies = sortBy(compose(toLower, prop('label')), technologies);
+  const sortedActiveTechnologies = sortedTechnologies.filter(({ inactive }) => inactive === false);
 
-  const handleScroll = (e: UIEvent<HTMLUListElement>) => {
+  const technologiesInUse = sortedActiveTechnologies.filter(({ ring }) => ring === TECHNOLOGY_RING.IN_USE);
+  const technologiesProven = sortedActiveTechnologies.filter(({ ring }) => ring === TECHNOLOGY_RING.PROVEN);
+  const technologiesPromising = sortedActiveTechnologies.filter(({ ring }) => ring === TECHNOLOGY_RING.PROMISING);
+  const technologiesPhasedOut = sortedActiveTechnologies.filter(({ ring }) => ring === TECHNOLOGY_RING.PHASED_OUT);
+
```

Comment:
How about writing some function like 
```js
getTechnologyByCategory = (tech: string) => sortedActiveTechnologies.filter(({ ring }) => ring === tech);
```
And use it like:
```js
const technologiesInUse = getTechnologyByCategory(TECHNOLOGY_RING.IN_USE);
const technologiesProven....
```

Maybe with better naming 😄

Reply:
Good point, done ✅